duckduckgo/iOS
# Update to Logins Prompts copy

Task/Issue URL: https://app.asana.com/0/0/1204587889053404/f
Tech Design URL:
CC:

**Description**:
Updates to Autofill Logins copy in preparation for Sync



**Steps to test this PR**:
1. With no logins saved, confirm the copy reads: `Logins are stored securely on your device.`
2. Go to a website e.g. `fill.dev` and attempt a login. Confirm the copy on the prompt to save the login reads: `Logins are stored securely on your device, and can be managed from the Logins menu in Settings.`
3. Review the translations confirming only autofill related strings have been updated



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
